lib/src/util/validationUtil.ts

```diff
@@ -11,7 +11,7 @@ export { ValidationErrors };
 /**
  * @return whether this ValidationErrors object contains an error
  */
-export const hasValidationErrors = (validationErrors: ValidationErrors | undefined) => {
+export const hasValidationErrors = (validationErrors: ValidationErrors) => {
```

Comment:
I suppose someone calling this from js instead of ts just needs to be aware that it will return true for undefined? Should the js doc spell out the accepted values for validation errors and/or be more explicit about the return type?

Reply:
> I suppose someone calling this from js instead of ts just needs to be aware that it will return true for undefined? Should the js doc spell out the accepted values for validation errors and/or be more explicit about the return type?

The change here has now been reverted, but at any rate, this function returns `false` for undefined.